File: Java/Easy/3110.java
class Solution {
    public int scoreOfString(String s) {
        int score = 0;
        int temp;
        for (int i = 0; i < s.length() - 1; i++) {
            temp = s.charAt(i) - s.charAt(i + 1);
            if (temp < 0) {
                temp *= -1;
            }

            score += temp;
        }

        return score;
    }
}

File: Python/Easy/3110.py
class Solution:
    def scoreOfString(self, s: str) -> int:
        score = 0

        for i in range(len(s) - 1):
            temp = abs(ord(s[i]) - ord(s[i + 1]))
            if temp < 0:
                temp *= -1
            score += temp

        return score
